modules/ztp-image-based-upgrade-concept.adoc

```diff
@@ -0,0 +1,147 @@
+// Module included in the following assemblies:
+// * edge_computing/image-based-upgrade/ztp-image-based-upgrade.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="ztp-image-based-upgrade-concept_{context}"]
+= Managing the image-based upgrade at scale using the ImageBasedGroupUpgrade CR on the hub
+
+The `ImageBasedGroupUpgrade` CR combines the `ImageBasedUpgrade` and `ClusterGroupUpgrade` APIs.
+For example, you can define the cluster selection and rollout strategy with the `ImageBasedGroupUpgrade` API in the same way as the `ClusterGroupUpgrade` API.
+The stage transitions are different from the `ImageBasedUpgrade` API.
+The `ImageBasedGroupUpgrade` API allows you to combine several stage transitions, also called actions, into one step that share a rollout strategy.
+
+.Example ImageBasedGroupUpgrade.yaml
+include::snippets/ibu-ImageBasedGroupUpgrade.adoc[]
+
+[id="ztp-image-based-upgrade-supported-combinations_{context}"]
+== Supported action combinations
+
+Actions are the list of stage transitions that {cgu-operator} completes in the steps of an upgrade plan for the selected group of clusters.
```

Comment:
```suggestion
Actions are the list of stage transitions that the {cgu-operator} completes in the steps of an upgrade plan for the selected group of clusters.
```

For consistency, as per https://github.com/openshift/openshift-docs/pull/86756/files#diff-31c7701b2c54c828ab2e3cbb1ca1b42dd00f088e36e57ad8a57cbe3c39978181R130